source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -16,18 +16,38 @@ oneDAL public identifiers are represented in the following header files:
      - A header file for a particular algorithm.
        The folder for the algorithm itself is ``oneapi/dal/algo/%ALGO%/``.
 
-       The string %ALGO% should be substituted with the name of the algorithm, for example, ``kmeans`` or ``knn``.
+       The string %ALGO% should be substituted with the name of the algorithm,
```

Comment:
minor: what was the reason of the changes which seem to be splitting one line in two?

Reply:
It's just an automatic tool that formates lines longer than 80 characters...